.github/workflows/continuous-integration.yml

```diff
@@ -1,8 +1,12 @@
 # https://help.github.com/en/categories/automating-your-workflow-with-github-actions
 
 on:
-  - pull_request
-  - push
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
```

Comment:
@Lctrs 

What do you think, does this makes sense when we want to build pushes for tags, regardless of the format?

Reply:
Should be `'**'` according to https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags.

Reply:
Thank you, @Lctrs!